pkg/proxy/winkernel/proxier.go

```diff
@@ -1131,7 +1138,7 @@ func (proxier *Proxier) syncProxyRules() {
 
 		hnsLoadBalancer, err := hns.getLoadBalancer(
 			hnsEndpoints,
-			loadBalancerFlags{isDSR: proxier.isDSR, isIPv6: proxier.isIPv6Mode, sessionAffinity: sessionAffinityClientIP},
+			loadBalancerFlags{isDSR: proxier.isDSR || svcInfo.localTrafficDSR, isIPv6: proxier.isIPv6Mode, sessionAffinity: sessionAffinityClientIP},
```

Comment:
We are not going to modify the Cluster IP loadbalancer since that is only accessible by internal traffic.

Reply:
gotcha, just updated the commit to leave cluster IP loadbalancer alone